backend/tracim_backend/models/context_models.py

```diff
@@ -1637,6 +1637,11 @@ def __init__(self, page: Page) -> None:
         self.items = page
 
 
+class ListItemsObject(object):
```

Comment:
Not sure how to name this, i do need to encapsulate in "items" my result.

Reply:
```suggestion
T = typing.TypeVar("T")


class ListItemsObject(typing.Generic[T]):
```